lib/matplotlib/tests/test_datetime.py

```diff
@@ -810,11 +810,32 @@ def test_triplot(self):
         fig, ax = plt.subplots()
         ax.triplot(...)
 
-    @pytest.mark.xfail(reason="Test for violin not written yet")
+    @pytest.mark.parametrize(("orientation"), ["vertical", "horizontal"])
```

Comment:
Superfluous parentheses.
```suggestion
    @pytest.mark.parametrize("orientation", ["vertical", "horizontal"])
```

Reply:
I've taken the liberty to commit this trivial change.